doc/api/models/schemas/RewardsInfo.yml

```diff
@@ -0,0 +1,43 @@
+type: object
+description: Information about rewards configuration on the network for calculating APY
+properties:
+  epochsPerYear:
+    type: integer
+    format: int64
+    description: Number of epochs per year (based on current GS epoch configuration)
+  currentDagPrice:
+    type: integer
+    format: int64
+    description: Current DAG price (in the same format as used internally in configuration)
```

Comment:
Same here

Reply:
updated to datum/usd with the assumption of the minimum supported price of 1 datum/usd. As above, I think we could revisit this with the updated types